File: en/java-spring-demo/src/main/java/com/itangcent/springboot/demo/common/dto/UserDto.java
package com.itangcent.springboot.demo.common.dto;

import com.itangcent.springboot.demo.common.model.UserInfo;
import org.springframework.web.multipart.MultipartFile;

/**
 * User Data Transfer Object
 * Extends UserInfo, adds file upload functionality
 * Used for handling user information forms that include file uploads
 */
public class UserDto extends UserInfo {
    /**
     * User Profile Image File
     * Used for uploading user's personal profile picture
     */
    private MultipartFile profileImg;

    public MultipartFile getProfileImg() {
        return profileImg;
    }

    public void setProfileImg(MultipartFile profileImg) {
        this.profileImg = profileImg;
    }
} 

File: en/kotlin-spring-demo/src/main/kotlin/com/itangcent/common/dto/UserDto.kt
package com.itangcent.common.dto

import com.itangcent.common.model.UserInfo
import org.springframework.web.multipart.MultipartFile

/**
 * User Data Transfer Object
 * Extends UserInfo, adds file upload functionality
 * Used for handling user information forms that include file uploads
 */
class UserDto : UserInfo() {
    /**
     * User Profile Image File
     * Used for uploading user's personal profile picture
     */
    var profileImg: MultipartFile? = null
}